stack/schema.go

```diff
@@ -7,13 +7,13 @@ package stack
 type Provider struct {
 	Name       string `yaml:"name"`
 	GatewayURL string `yaml:"gateway"`
-	Network    string `yaml:"network"`
+	Network    string `yaml:"network,omitempty"`
 }
 
 // Function as deployed or built on FaaS
 type Function struct {
 	// Name of deployed function
-	Name     string `yaml:"-"`
+	Name     string `yaml:"-,omitempty"`
```

Comment:
Does this re-serialize correctly? I had some issues getting this to map back in the correct way.

Reply:
That's strange, what kind of issues did you experience? The `omitempty` is there so that if the `Name` property is left out of the `Function` struct instantiation, it will not be Marshalled to YAML. Without `omitempty`, the YAML file will end up with a `name: ""` property.